Userland/Utilities/pgrep.cpp

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (c) 2021, Aziz Berkay Yesilyurt <abyesilyurt@gmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <AK/QuickSort.h>
+#include <AK/Vector.h>
+#include <LibCore/ArgsParser.h>
+#include <LibCore/ProcessStatisticsReader.h>
+#include <LibRegex/Regex.h>
+
+int main(int argc, char** argv)
+{
+    if (pledge("stdio rpath", nullptr) < 0) {
+        perror("pledge");
+        return 1;
+    }
+
+    bool case_insensitive = false;
+    bool invert_match = false;
+    const char* pattern = nullptr;
```

Comment:
Please use east-const (as per the project code style)

Reply:
Done!